app/services/search/chat_channel_membership.rb

```diff
@@ -7,6 +7,14 @@ class ChatChannelMembership < Base
     DEFAULT_PER_PAGE = 30
 
     class << self
+      def search_documents(params:)
```

Comment:
This is in the `search/base.rb` class so you shouldn't need to define this method again here.